apache/superset
# fix: weight tooltip issue

### SUMMARY

Screen grid visualization shows Weight undefined

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

BEFORE:
<img width="730" alt="image" src="https://user-images.githubusercontent.com/39701522/160450942-d774332f-57e8-4261-a79d-95e774e88cfe.png">

AFTER:
<img width="750" alt="image" src="https://user-images.githubusercontent.com/39701522/160450676-ec165513-6632-4ac6-84e1-717bc31a6a4a.png">

### TESTING INSTRUCTIONS

1. Go to chart list
2. Select send grid chart.
3. See if weight is undefined or not in tooltip.

### ADDITIONAL INFORMATION
